.github/workflows/build.yml

```diff
@@ -38,6 +35,6 @@ jobs:
       - name: Build production image
         uses: docker/build-push-action@v3
         with:
-          push: true
+          push: ${{ github.event_name != 'pull_request' }}
```

Comment:
To prevent this from triggering when someone does a direct push to main, shouldn't we be using `github.event.pull_request.merged == true` event instead as the condition ?

Reply:
It makes sense, yes.